src/DIRAC/Core/Base/Client.py

```diff
@@ -3,8 +3,19 @@
     This class exposes possible RPC calls, given a url of a service.
 """
 import ast
-import importlib_resources
-from functools import partial, partialmethod
+from functools import partial
+from importlib import resources
+
+try:
+    from functools import partialmethod
+except ImportError:
+
+    class partialmethod(partial):
+        def __get__(self, instance, owner):
+            if instance is None:
+                return self
+            return partial(self.func, instance, *(self.args or ()), **(self.keywords or {}))
```

Comment:
I think this is from a bad rebase as this was a Python 2 compaibility hack that was removed from 8.x in ce39d7e805f40e5fc28407f7fa1573a44661efff.